README.md

```diff
@@ -1,31 +1,46 @@
 
-### Important differences
+### Important ApostropheCMS differences
 Where this project differs from a normal ApostropheCMS project is that no frontend code should be included in your modules. So, client-side JavaScript and styling, normally added to the `modules/custom-module/ui/src` folder will now be included in the Astro project. This also includes templates in the `views` folder of most modules. One exception is the `modules/@apostrophecms/home-page` module. This module provides a "fall-back" for users who navigate to the Apostrophe server (located by default at `localhost:3000` during development) and simply loads the core `views/layout.html` file. This core file has been modified in this project to provide info about the project status and not load any of the ApostropheCMS Admin UI.
 
 Equally, certain [module customization functions](https://docs.apostrophecms.org/reference/module-api/module-overview.html#customization-functions) that deal with front-end functionality should not be used. This includes the `helper()` and `extendHelpers()` functions for providing Nunjucks template helpers, the `components()` method that provide asynchronous template components, and the `renderRoutes()` function to return a rendered template.
 
+An advantage of using Astro as a frontend unlike an ApostrophCMS-only project, is that the templates for widgets can be used as templates outside specialized `area` schema fields. While this removes the ability to edit the widgets in-context, it does provide flexibility for page design without code repetition. However, this means that the widget schema needs to be added to the schema fields of the page. To facilitate this, some of the widget schema fields have been moved to the `lib/schema-mixins` folder. This allows them to be imported and used in both the main widget module and in the home page template.
+
+### Similarities to a stand-alone Astro project
+The Astro half of this project has standard components and templates.
+
+### Important Astro differences
+Unlike an Astro project with multiples routes set-up in the `pages` folder, this project has a single `[...slug].astro` to that handles all the routing using pages mapped to the `templates` folder. Each of the templates corresponds to one of the registered ApostropheCMS page or piece-page types. The content of these templates is populated by data from the CMS backend and is added into the slots in the `[...slug].astro` file. There is also a folder containing templates for ApostropheCMs widgets. The ApostropheCMS page and widget types are mapped to the templates through an `index.js` page in the corresponding folders.
```

Comment:
multiples -> multiple

maybe just remove "set-up"?

"to that handles" -> "that handles"

ApostropheCMs -> ApostropheCMS

Reply:
`index.js` file?

Reply:
Fixed and changed the text.

Reply:
Changed the language for clarity, as well.